subsys/bluetooth/mesh/lpn.c

```diff
@@ -284,6 +284,10 @@ static void friend_req_sent(uint16_t duration, int err, void *user_data)
 		return;
 	}
 
+	if (lpn->state == BT_MESH_LPN_DISABLED) {
```

Comment:
Could even be `lpn->state != BT_MESH_LPN_ENABLED`, no?

Reply:
Yes, if this is the only valid state that the LPN should be in for the CB to complete, it gives more sense. Must confess that I'm not sure if there is any other possible state that is valid, but if you are quite certain then I will change it.